build-tools-internal/src/main/resources/templates/migration-index.asciidoc

```diff
@@ -0,0 +1,35 @@
+[[breaking-changes]]
+= Migration guide
```

Comment:
Perhaps it would make more sense to have this template file contain nothing but the dynamic bits at the bottom, together with an include directive for the main content which should be in the docs section, and not in the template, for easier editing.